dist/index.js

```diff
@@ -46049,6 +46049,9 @@ var Status;
 })(Status || (Status = {}));
 // run executes the buf command with the given arguments.
 async function run(bufPath, args) {
+    if (core.isDebug()) {
```

Comment:
Hmm, so this is already defined? Is there any work needed to expose it in the config?

Reply:
Sorry I missed the description